pattoo_shared/encrypt.py

```diff
@@ -214,6 +215,10 @@ def exp_main(self, _wrapper='(main)'):
                                 .format(_filename, _wrapper)))
         with open('{}'.format(_file), '{}'.format('w')) as f:
```

Comment:
You are reusing the 'f' variable name, once as a filename and then as a file handle. This could cause problems later with editing. This is not a good coding practice. Used dedicated variable names if possible. Reuse in dedicated loops is better, but not ideal. Please fix.